templates/remote_interact_base.rs.html

```diff
@@ -25,6 +25,7 @@ <h2>@i18n!(ctx.1, "I'm from this instance")</h2>
 	    			.default(login_form.password)
 	    			.error(&login_errs)
 	    			.set_prop("minlength", 1)
+				.input_type("password")
```

Comment:
Just a little alignment issue
```suggestion
				    .input_type("password")
```